.github/ISSUE_TEMPLATE/bump_gmt_checklist.md

```diff
@@ -0,0 +1,29 @@
+---
+name: Bump GMT version checklist
+about: Checklist for bumping the minimum required GMT version.
+title: Bump to GMT X.Y.Z
+labels: maintenance
+assignees: ''
+
+---
+
+:tada: [GMT X.Y.Z](https://github.com/GenericMappingTools/gmt/releases/tag/X.Y.Z) has been released! It is installable from the
+[conda-forge channel](https://anaconda.org/conda-forge/gmt/files) using the following command:
+
+`conda install -c conda-forge gmt=X.Y.Z`
+
+<!-- Please add specific checklist items for the tests, xfail pytest markers, and deprecated syntax that need to be updated. -->
+
+**To-Do**:
+- [ ] Bump the minimum required GMT version (1 PR)
```

Comment:
Why is 1 PR specified? Would we expect it to be a single PR if this is a PR template?

Reply:
I based this off https://github.com/GenericMappingTools/pygmt/issues/1320 and https://github.com/GenericMappingTools/pygmt/pull/1321. Since the updates in the checkboxes under this item are related, I expect they would be easier to review as one PR.